cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -1007,15 +1013,15 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
           }
         }
       } catch (Throwable t) {
-        LOG.info("InputFormat class not found: {}", t.getMessage(), t);
+        LOG.error("InputFormat class not found: {}", t.getMessage(), t);
```

Comment:
If the exception is ignored, why logging it as an error? I think it should be logged as debug. We ignore this because even if it is not able to find the inputformat class, the job might still be able to run (e.g. a different classpath being configured for the MR task containers that actually has the input format class).